Samples/README.md

```diff
@@ -1,13 +1,131 @@
+
+        cd bearer-token-generator
+3. Initialize npm
+
+        npm init
+4. Install `skyflow-node`
+
+        npm i skyflow-node
+5. Create `index.js` file
+6. Open `index.js` file
+7. populate `index.js` file with below code snippet
+```javascript
+const express = require('express')
+const app = express()
+var cors = require('cors')
+const port = 3000
+const {
+    generateBearerToken,
+    isExpired
+} = require('skyflow-node');
+
+app.use(cors())
+
+let filepath = 'cred.json';
+let bearerToken = "";
+
+function getSkyflowBearerToken() {
+    return new Promise(async (resolve, reject) => {
+        try {
+            if (!isExpired(bearerToken)) resolve(bearerToken)
+            else {
+                let response = await generateBearerToken(filepath);
+                bearerToken = response.accessToken;
+                resolve(bearerToken);
+            }
+        } catch (e) {
+            reject(e);
+        }
+    });
+}
+
+app.get('/', async (req, res) => {
+  let bearerToken = await getSkyflowBearerToken();
+  res.json({"accessToken" : bearerToken});
+})
+
+app.listen(port, () => {
+  console.log(`Server is listening on port ${port}`)
+})
+```
+8. Start the server
+
+        node index.js
+    server will start at `localhost:3000`
+9. Your **<TOKEN_END_POINT_URL>** with `http://localhost:3000/`
+
+## The samples
+### Collect and reveal
+This sample illustrates how to use secure Skyflow elements to collect sensitive user information and reveal it to the user via tokens.
+#### Configure
+1. In `Skyflow.Configuration()` of [ViewController.swift](CollectAndRevealSample/CollectAndRevealSample/ViewController.swift), replace with the following fields:
+         - Replace the placeholder "<VAULT_ID>" with the correct vaultId you want to connect
+         - Replace the placeholder "<VAULT_URL>" with the correct vaultURL
+2. Update `Fields` struct in [ResponseStructs.swift](CollectAndRevealSample/CollectAndRevealSample/ResponseStructs.swift) with field name of used vault. For ex: 
+        
+        ```swift
+            struct Fields: Codable {
+               let name: NameField
+               let cvv: String
+               let cardExpiration: String
+               let cardNumber: String
+               let skyflow_id: String
+            }
+        ```
+        
+    The fields can be different depending upon the vault.
+3. Update 
+4. Replace the placeholder "<TOKEN_END_POINT_URL>" of [ExampleTokenProvider.swift](CollectAndRevealSample/CollectAndRevealSample/ExampleTokenProvider.swift) with the  bearer token endpoint which gives the bearerToken, implemented at your backend or `http://localhost:3000/`.
+ 5. Running the sample
+    1. Open CMD
+    2. Navigate to `CollectAndRevealSample`
+    3. Run 
+
+            pod install
+    4. Open the [`CollectAndRevealSample.xcworkspace](`CollectAndRevealSample/`CollectAndRevealSample.xcworkspace) file using xcode
+    5. click on build and run
```

Comment:
I couldn't get this running. Also, Xcode doesn't have a button with the text "build and run" that I could find.

Reply:
`build and run` in xcode signifies to a triangular shaped button, when we hover over it, it mentions `Build and then run the current schema`.